lib/fathead/python_six/output.txt

```diff
@@ -0,0 +1,142 @@
+PY2	A										<section class="prog__container"><p>A boolean indicating if the code is running on Python 2.</p><pre><code></code></pre></section>	https://pythonhosted.org/six//index.html
```

Comment:
@iamrajhans looks like this has an empty `<pre><code>` ? if there's no code, we shouldn't add the codeblock to the abstract